runtime/mandala/src/benchmarking/honzon.rs

```diff
@@ -275,15 +212,10 @@ runtime_benchmarks! {
 		let debit_amount: Amount = debit_amount.unique_saturated_into();
 		let collateral_value = 10 * debit_value;
 		let collateral_amount = Price::saturating_from_rational(dollar(currency_id), dollar(STABLECOIN)).saturating_mul_int(collateral_value);
-		// set balance and trading path
-		set_balance(currency_id, &sender, (10 * collateral_amount) + ExistentialDeposits::get(&currency_id));
 
-		default_path.insert(0, currency_id);
-		for i in 0..default_path.len() {
-			if i != 0 {
-				inject_liquidity(maker.clone(), default_path[i], default_path[i-1], 10_000 * dollar(default_path[i]), 10_000 * dollar(default_path[i-1]), false)?;
-			}
-		}
+		// set balance and inject liquidity
+		set_balance(currency_id, &sender, (10 * collateral_amount) + ExistentialDeposits::get(&currency_id));
+		inject_liquidity(maker.clone(), currency_id, STABLECOIN, 10_000 * dollar(currency_id), 10_000 * dollar(STABLECOIN), false)?;
```

Comment:
This makes it so the swap path is LIQUID => STABLECOIN in benchmark, when the worst case for acala/karura is LIQUID => STAKING => STABLECOIN. that will remove 2 read and 1 write from weights